zkevm-circuits/src/util/word.rs

```diff
@@ -231,6 +364,11 @@ impl<F: Field> Word<Expression<F>> {
     pub fn sub_unchecked(self, rhs: Self) -> Self {
         Word::new([self.lo() - rhs.lo(), self.hi() - rhs.hi()])
     }
+
+    /// No overflow check
+    pub fn expr_unchecked(&self) -> Expression<F> {
```

Comment:
```suggestion
    /// Compress the lo and hi limbs into an expression without checking the overflow.
    /// Only use it for address. We should remove it before merging to the main branch.
    #[deprecated]
    pub fn expr_unchecked(&self) -> Expression<F> {
```

Reply:
cosmetics comment a bit and add deprecated msg in commit https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1441/commits/84f96c38881e5cc4000c166e314242be80639c11 👌